hw1-app/src/app/shared/shared.module.ts

```diff
@@ -1,8 +1,9 @@
 import { NgModule } from '@angular/core';
+import { FormsModule }   from '@angular/forms';
 import { SearchComponent } from './search/search.component';
 
 @NgModule({
-  imports: [],
+  imports: [FormsModule],
   declarations: [SearchComponent],
   exports: [SearchComponent],
```

Comment:
Why do you need exports here

Reply:
Because we use this component in other modules